src/classes/OPP_OpportunityContactRoles_TDTM.cls

```diff
@@ -730,9 +741,12 @@ public without sharing class OPP_OpportunityContactRoles_TDTM extends TDTM_Runna
         }
 
         for (Opportunity eachOppty : currentOpportunities) {
-            if (eachOppty.Primary_Contact__c != null && eachOppty.AccountId != null
+            if (eachOppty.Primary_Contact__c != null 
```

Comment:
Would this method be easier to read if the check for `eachOppty.DisableContactRoleAutomation__c` was after the for using a `continue'`

Reply:
The check for `Disable Contact Role Automation` is another check as verification the Primary Contact is not null, so I would treat them the same. Please let me know if that is not the case.

Reply:
I think it's ok to add it to the existing conditional statement and not having a separate check, I agree that it is definitely hard to follow along the large conditional but that can be addressed in a separate refactor.